### PR TITLE
docs: update description of how to join slack

### PR DESCRIPTION
**Pull request by juzhiyuan**
### Description

This PR updates description of how to join the Slack channel, as many community users complain that it's hard to find the correct channel after joining. I choose to link it to the website description.

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [x] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

**Review by SylviaBABY — APPROVED** [review]

**Review by hf400159 — APPROVED** [review]

**Comment by Canonical10** [comment]
✅

**Review by Canonical10 — APPROVED** [review]
✅

**Review by Canonical10 — APPROVED** [review]
✅

**Review by spacewander — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ The technical architecture of Apache APISIX:
 
 - Mailing List: Mail to dev-subscribe@apisix.apache.org, follow the reply to subscribe to the mailing list.
 - QQ Group - 552030619, 781365357
-- Slack Workspace - join [`#apisix` channel](https://join.slack.com/t/the-asf/shared_invite/zt-vlfbf7ch-HkbNHiU_uDlcH_RvaHv9gQ) on ASF Slack to meet the team and ask questions, or send `Request to join APISIX slack` mail to the mail list(dev@apisix.apache.org), we will invite you in.
+- Slack Workspace - join [`#apisix` channel](https://apisix.apache.org/docs/general/join/#join-the-slack-channel)
```
Comment (moonming) [inline README.md:46]: @juzhiyuan Instead of jumping again, you can paste content directly here. One more jump, one more loss

**Pull request merged**